index.js

```diff
@@ -1,80 +1,93 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] it would be better if you fix your code and remove the eslint disable comment cause we are setting the eslint in order to write a clean and professional  code

Reply:
Because we distributed the files, some variables are now used before being defined in some of them. Therefore we still see ESLint issues, so we still added some eslint-disable commands.